lib/Resource.js

```diff
@@ -21,7 +23,7 @@ class Resource {
 	 *					(cannot be used in conjunction with parameters buffer, stream or createStream)
 	 * @param {Stream} [parameters.stream] Readable stream of the content of this resource
 	 *					(cannot be used in conjunction with parameters buffer, string or createStream)
-	 * @param {function} [parameters.createStream] Function callback that returns a readable stream of the content
+	 * @param {Function} [parameters.createStream] Function callback that returns a readable stream of the content
```

Comment:
I would prefer lowercase `function`. At least in UI5, we have ~2500 lowercase vs ~500 Uppercase.